scripts/view/projectView.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+(function(module) {
+
+	var projectView = { };
+
+  projectView.renderProjects = function() {
+    Project.allProjects.forEach(function(projectInstance){
+      $('#projects').append(projectInstance.compileTemplate());
+      if($('#type-filter option:contains("' + projectInstance.projectType + '")').length === 0) {
+        $('#type-filter').append('<option>' + projectInstance.projectType + '</option>');
+      }
+    })
+  };
```

Comment:
You are trying to use the value set on an option to trigger a URL redirect with Page.js in projectView.projectTypeFilter. Currently, your options have no value! You could fix that easily in your append statement:
```
projectView.renderProjects = function() {
    Project.allProjects.forEach(function(projectInstance){
      $('#projects').append(projectInstance.compileTemplate());
      if($('#type-filter option:contains("' + projectInstance.projectType + '")').length === 0) {
        $('#type-filter').append(`<option value="${projectInstance.projectType}">${projectInstance.projectType}</option>`);
      }
    })
  };
```